test/jwt.asymmetric_signing.tests.js

```diff
@@ -73,21 +78,21 @@ describe('Asymmetric Algorithms', function(){
 
         context('synchronous', function () {
           it('should validate with public key', function () {
-            var decoded = jwt.verify(token, pub);
+            const decoded = jwt.verify(token, pub);
             assert.ok(decoded.foo);
             assert.equal('bar', decoded.foo);
           });
 
           it('should throw with invalid public key', function () {
-            var jwtVerify = jwt.verify.bind(null, token, invalid_pub)
+            const jwtVerify = jwt.verify.bind(null, token, invalid_pub)
             assert.throw(jwtVerify, 'invalid signature');
           });
         });
 
       });
 
       describe('when signing a token with expiration', function () {
-        var token = jwt.sign({ foo: 'bar' }, priv, { algorithm: algorithm, expiresIn: '10m' });
+        let token = jwt.sign({ foo: 'bar' }, priv, { algorithm: algorithm, expiresIn: '10m' });
```

Comment:
Yes, I tried to make everything const but it failed because another test was reassigning the variable. I checked again and the problem is that this variable was in he wrong scope. I moved it and now it works with const 🙂